wwwapp/views.py

```diff
@@ -385,7 +387,8 @@ def participants_view(request, year):
                 'accepted_workshop_count': 0,
                 'workshop_count': 0,
                 'has_letter': bool(cover_letter and len(cover_letter) > 50),
-                'status': participant.participant.status_for(year),
+                'status': workshop_profile.status if workshop_profile else None,
+                'status_display': workshop_profile.get_status_display if workshop_profile else None,
```

Comment:
Ohh I forgot this view does this stupid copy-paste-all-the-things so you can't access get_status_display but fine, let's leave it like this :P